extensions/arc/src/ui/dashboards/controller/controllerDashboardOverviewPage.ts

```diff
@@ -218,16 +218,26 @@ export class ControllerDashboardOverviewPage extends DashboardPage {
 						iconHeight: iconSize,
 						iconWidth: iconSize
 					}).component();
-				const nameLink = this.modelView.modelBuilder.hyperlink()
-					.withProperties<azdata.HyperlinkComponentProperties>({
-						label: r.instanceName || '',
-						url: ''
-					}).component();
-				nameLink.onDidClick(async () => {
-					await this._controllerModel.treeDataProvider.openResourceDashboard(this._controllerModel, r.instanceType || '', parseInstanceName(r.instanceName));
-				});
+				let nameComponent: azdata.Component;
+				if (r.instanceType === ResourceType.postgresInstances) {
+					nameComponent = this.modelView.modelBuilder.text()
+						.withProperties<azdata.TextComponentProperties>({
+							value: r.instanceName || '',
+							CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
+						}).component();
+				} else {
+					nameComponent = this.modelView.modelBuilder.hyperlink()
+						.withProperties<azdata.HyperlinkComponentProperties>({
+							label: r.instanceName || '',
+							url: ''
+						}).component();
+					(<azdata.HyperlinkComponent>nameComponent).onDidClick(async () => {
+						await this._controllerModel.treeDataProvider.openResourceDashboard(this._controllerModel, r.instanceType || '', parseInstanceName(r.instanceName));
+					});
+				}
+
 				// TODO chgagnon
```

Comment:
If this TODO is still valid, then can you please add a one-liner on what remains to be done.